tests/functional/a/arguments_differ.py

```diff
@@ -1,5 +1,6 @@
 """Test that we are emitting arguments-differ when the arguments are different."""
-# pylint: disable=missing-docstring, too-few-public-methods, unused-argument,useless-super-delegation, unused-private-member
+# pylint: disable=missing-docstring, too-few-public-methods, unused-argument,useless-super-delegation
+# pylint: disable=unused-private-member, bad-dunder-name
 
 class Parent:
 

```

Comment:
```suggestion

```
To avoid the full result of the functional test being changed because of the line change.